pkg/morph/client/notary.go

```diff
@@ -516,42 +519,34 @@ func (c *Client) notaryInvoke(committee, invokedByAlpha bool, contract util.Uint
 		Signers: cosigners,
 	}
 
-	// calculate notary fee
-	notaryFee, err := c.client.CalculateNotaryFee(u8n)
+	// define witnesses
+	mainTx.Scripts = c.notaryWitnesses(invokedByAlpha, multiaddrAccount, mainTx)
```

Comment:
Consider using notary actor as `nAct.Notarize(nAct.MakeCall(contract, method, params...))`, this should simplify the code above (most of it is just not needed).